.tekton/jupyter-minimal-ubi9-python-3-11-pull-request.yaml

```diff
@@ -10,7 +10,11 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/cancel-in-progress: "true"
     pipelinesascode.tekton.dev/max-keep-runs: '3'
-    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && has(body.repository) && body.repository.full_name == "opendatahub-io/notebooks"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( "jupyter/minimal/ubi9-python-3.11/Pipfile.lock".pathChanged()
+      || "jupyter/minimal/ubi9-python-3.11/start-notebook.sh".pathChanged() || "jupyter/utils/***".pathChanged()
+      || ".tekton/jupyter-minimal-ubi9-python-3-11-pull-request.yaml".pathChanged()
+      || "jupyter/minimal/ubi9-python-3.11/Dockerfile.cpu".pathChanged() ) && has(body.repository)
```

Comment:
Just wondering - why not simply use `jupyter/minimal/ubi9-python-3.11/***`? :thinking:

Reply:
This way, it's possible to make change in .cpu Dockerfile without triggering a .cuda Dockerfile rebuild

You're right that quite possibly it's not worth it, given how long these yaml conditions become.

I'll change to your suggestion.

Reply:
Ah, you're right. I missed the cpu vs cuda thing. So, if we are sure we'll have these expressions always up-to-date (generate_code.sh as you mention below), then I'm fine with this.

Both way (the current and the simplistic ***) are okay for me right now when condition above is fullfiled.